integ/src/main.rs

```diff
@@ -0,0 +1,277 @@
+use log::info;
+use snafu::ResultExt;
+use std::env::temp_dir;
+use std::fs;
+use std::{env, process};
+use tokio::time::{sleep, Duration};
+
+use integ::ec2_provider::{create_ec2_instance, terminate_ec2_instance};
+use integ::eks_provider::{get_cluster_info, write_kubeconfig};
+use integ::error::ProviderError;
+use integ::updater::{delete_cluster_resources, label_node, run_brupop};
+
+type Result<T> = std::result::Result<T, error::Error>;
+
+/// The default path for kubeconfig file
```

Comment:
It might be ideal to place the file in [`std::env::temp_dir`](https://doc.rust-lang.org/std/env/fn.temp_dir.html) and output the final file location to stdout. This would allow callers to overwrite the location of the file by setting TEMPDIR.